drgn/cli.py

```diff
@@ -182,6 +202,16 @@ def _main() -> None:
             prog.set_core_dump(args.core)
         elif args.pid is not None:
             prog.set_pid(args.pid or os.getpid())
+        elif args.openocd:
+            if args.openocd_tap is None:
+              print("error: --openocd-tap required when using OpenOCD", file=sys.stderr)
+              sys.exit(1)
```

Comment:
Tiny suggestion, Python allows you to shorten this:
```python
              sys.exit("error: --openocd-tap required when using OpenOCD")
```
Non-int args are printed to stderr and an error code of 1 is returned.

Reply:
Ack, I'll take care of that when rebasing.